models/src/main/scala/it/pagopa/interop/catalogmanagement/model/CatalogDescriptor.scala

```diff
@@ -12,5 +12,6 @@ final case class CatalogDescriptor(
   audience: Seq[String],
   voucherLifespan: Int,
   dailyCallsPerConsumer: Int,
-  dailyCallsTotal: Int
+  dailyCallsTotal: Int,
+  agreementApprovalPolicy: Option[PersistentAgreementApprovalPolicy]
```

Comment:
Why optional?

Reply:
for backward compatibility.
I guess these classes will be used also to deserialize data from read-model, and for existing descriptors the new field does not exist.
BTW, this field is mandatory in API interface, so new elements will always have it populated